pkg/apis/velero/v1/schedule.go

```diff
@@ -77,8 +77,18 @@ type ScheduleStatus struct {
 	ValidationErrors []string `json:"validationErrors,omitempty"`
```

Comment:
For consistency with the scaffolding code generated by kubebuilder, it'd better be renamed to xxx_type.go so it's more clearly identified.